lib/virtio/virtqueue.c

```diff
@@ -586,6 +681,12 @@ static int vq_ring_enable_interrupt(struct virtqueue *vq, uint16_t ndesc)
 #endif /*VIRTIO_MASTER_ONLY*/
 	}
 
+#ifdef VIRTIO_D_CACHE
+	metal_cache_flush(VRING_USED(vq), vring_used_size(VRING_NUM(vq)));
```

Comment:
`vq->vq_ring.used->flags` could be updated here by slave and it could be read by master in `vq_ring_must_notify()`
`vq->vq_ring.avail->flags` could be updated here by master and it could be read by slave in `vq_ring_must_notify()`

Reply:
Seems that I have  some comments are not at the good place...need to resetrt the review

Reply:
Wait for my next revision before restarting the review, I'm doing a bit of cleanup.